### PR TITLE
notepadqq: update to 1.4.4

### PR DESCRIPTION
**Pull request by abenson**

### CODE REVIEW

File: srcpkgs/notepadqq/template [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 build_style=qmake
 hostmakedepends="pkg-config qt5-tools"
 makedepends="qt5-svg-devel qt5-webkit-devel"
 depends="desktop-file-utils hicolor-icon-theme"
-maintainer="Orphaned <orphan@voidlinux.eu>"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
 short_desc="Text editor for developers"
 homepage="https://notepadqq.com"
-license="GPL-3"
+license="GPL-3.0"
```
Comment (maxice8) [inline srcpkgs/notepadqq/template]: needs -only or -or-later identifier

**Comment by abenson** [comment]
Sorry, had a brainfart and used [this](https://spdx.org/licenses/GPL-3.0.html) as the reference.

**Comment by cr6git** [comment]
[#14056](https://github.com/voidlinux/void-packages/pull/14056)

@abenson You're not paying attention. According to [notepadqq/releases](https://github.com/notepadqq/notepadqq/releases) the latest is still 1.4.0.

**Comment by abenson** [comment]
hm. okay. weird.

**Pull request closed without merging**